source/beamline/BioXAS/BioXASSideCryostatStage.cpp

```diff
@@ -1,19 +1,17 @@
 #include "BioXASSideCryostatStage.h"
 
-BioXASSideCryostatStage::BioXASSideCryostatStage(QObject *parent) :
-	BioXASCryostatStage("SideCryostatStage", parent)
+BioXASSideCryostatStage::BioXASSideCryostatStage(const QString &deviceName, QObject *parent) :
+	BioXASCryostatStage(deviceName, parent)
 {
-	cryostatX_ = new CLSMAXvMotor("SideCryostatX", "SMTR1607-6-I22-14", "SideCryostatX", true, 0.01, 2.0, this, ":mm");
-	cryostatY_ = new CLSMAXvMotor("SideCryostatY", "SMTR1607-6-I22-13", "SideCryostatY", true, 0.01, 2.0, this, ":mm");
-	cryostatZ_ = new CLSMAXvMotor("SideCryostatZ", "SMTR1607-6-I22-15", "SideCryostatZ", true, 0.01, 2.0, this, ":mm");
-
+	cryostatX_ = new CLSMAXvMotor(name()+"CryostatX", "SMTR1607-6-I22-14", name()+"CryostatX", false, 0.01, 2.0, this, ":mm");
```

Comment:
Due to the simplicity of these append calls, it's alright, but I'd prefer if we use the QStringBuilder `%` or using QString::arg() methods because they are more efficient.